package.json

```diff
@@ -27,12 +27,9 @@
   "devDependencies": {
     "@changesets/changelog-github": "^0.5.0",
     "@changesets/cli": "^2.27.5",
-    "@replay-cli/pkg-build": "workspace:^",
     "@replayio/protocol": "^0.73.0",
-    "@replayio/replay": "workspace:^",
     "@types/ws": "^8",
     "prettier": "^2.7.1",
-    "replayio": "workspace:^",
```

Comment:
It turns out that dependencies of the root are treated like global dependencies, they invalidate everything. So before this PR each change to `replayio`'s source code caused *every* other package to be invalidated because the root depended on it.